LetsDinnerV2 MessagesExtension/Model/Event.swift

```diff
@@ -177,7 +177,8 @@ class Event {
                 
                   let parameters : [String : Any] = ["title" : recipe.title ?? "",
                                                      "sourceUrl" : recipe.sourceUrl ?? "",
-                                                    "customOrder":recipe.customOrder]
+                                                     "id" : recipe.id ?? 0,
```

Comment:
Default value should be a unique number, UUID.uuidString()

Reply:
I think the default value is not necessary right? It is not useful even we have an unique ID. 
Its just for solving the optional type. How about guard let the recipe.id?

Reply:
But then if you do guard let and it has no id it will return without completing the function

Reply:
The id is for matching with the id which saved in customOrder. 
If the id is a default value, they wont be matched. That means the id is not useful if it is a default value. 
So, it doesn't matter what the default value is.

Reply:
Alright